CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+- fix: Transition from apiextensions.k8s.io/v1beta1 to
```

Comment:
I'd say this was a feature rather than a fix btw.

Reply:
Fixed.